data-loader/build.gradle

```diff
@@ -1,9 +1,11 @@
 subprojects {
     group = "scalardb.dataloader"
 
+
     ext {
         apacheCommonsLangVersion = '3.14.0'
         apacheCommonsIoVersion = '2.16.1'
+        lombokVersion = '1.18.34'
```

Comment:
I noticed `ext` block that defines dependency versions is written in the root `build.gradle`. How about moving this block into it? It has `commonsLangVersion = '3.14.0'` always.

Reply:
Good point. I will use the `commonsLangVersions`. That said, I added the ext in this build.gradle file to separate some dependencies that only apply to the data-loader subprojects. I don't think Lombok will be used in other (sub)projects.